tests/test_logging.py

```diff
@@ -538,6 +575,28 @@ def test_does_not_send_logs_outside_of_run_context_with_ignore_setting(
         output = capsys.readouterr()
         assert output.err == ""
 
+    def test_does_not_raise_or_warn_when_logger_outside_of_run_context_with_ignore_setting(
+        self,
+        logger,
+    ):
+        with temporary_settings(
+            updates={PREFECT_LOGGING_ORION_WHEN_MISSING_FLOW: "ignore"},
+        ):
+            try:
+                logger.info("test")
+            except TypeError:
+                pytest.fail(
+                    "Raised TypeError when logging outside of context with `ignore` setting."
+                )
+            except UserWarning:
+                pytest.fail(
+                    "Raised UserWarning when logging outside of context with `ignore` setting."
+                )
+            except Exception:
+                pytest.fail(
+                    "Raised unexpected exception when logging outside of context with `ignore` setting."
+                )
```

Comment:
No need for all these, warnings will raise as errors during testing
```suggestion
            logger.info("test")
```